penrose-web/src/__tests__/mathtransform.test.ts

```diff
@@ -0,0 +1,5 @@
+// import defineTest from 'jscodeshift/dist/testUtils';
```

Comment:
`import { defineTest } from "jscodeshift/dist/testUtils";` seems to work for me. Basically, you can use `import` syntax to import a particular exported member of a module. The line commented out imports the __default__ exported member instead.

Reply:
It works now! Thanks

Reply:
BTW, `jest` doesn't really like subpath imports like `jscodeshift/dist/testUtils`. Is `defineTest` exported by just `jscodeshift`? If so, you might need to do that instead to let CI pass. It's currently failing. See: https://app.circleci.com/pipelines/github/penrose/penrose/1548/workflows/e71b9cc2-90d4-4dac-89de-17d3ed29b5d8/jobs/3546/parallel-runs/0/steps/0-105

Reply:
After reading the README, seems like it works after I did `yarn add jscodeshift` in 7c2a9b9! This branch seems to be good to go for me!